cmd/influx/main.go

```diff
@@ -35,15 +35,15 @@ func init() {
 }
 
 var (
-	tokenFlag       = "token"
```

Comment:
Renamed these for clarity since I needed to make a top-level variable with type `cli.Flag`